index.html

```diff
@@ -0,0 +1,260 @@
+                width="560"
+                height="315"
+                src="https://www.youtube.com/embed/8mceJBrgUkQ"
+                title="YouTube video player"
+                frameborder="0"
+                allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+                allowfullscreen
+              ></iframe>
+            </div>
+            <div class="details">
+              <a
+                href="https://www.youtube.com/watch?v=8mceJBrgUkQ"
+                target="_blank"
+                ><p class="details-title">From Microverse to Microsoft</p></a
+              >
+              <p class="small">Microverse</p>
+              <div class="views">
+                <p>11k views</p>
+                <p>· 2 years ago</p>
+              </div>
+            </div>
+          </div>
+
+          <div class="content-details">
+            <div class="video">
+              <iframe
+                width="560"
+                height="315"
+                src="https://www.youtube.com/embed/KmsrbokC6kI"
+                title="YouTube video player"
+                frameborder="0"
+                allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+                allowfullscreen
+              ></iframe>
+            </div>
+            <div class="details">
+              <a
+                href="https://www.youtube.com/watch?v=KmsrbokC6kI"
+                target="_blank"
+              >
+                <p class="details-title">
+                  From Bartender to Front-end Engineer in less than a year
+                </p>
+              </a>
+              <p class="small">Microverse</p>
+              <div class="views">
+                <p>2.3k views</p>
+                <p>· 5 months ago</p>
+              </div>
+            </div>
+          </div>
+
+          <div class="content-details">
+            <div class="video">
+              <iframe
+                width="560"
+                height="315"
+                src="https://www.youtube.com/embed/jS4aFq5-91M"
+                title="YouTube video player"
+                frameborder="0"
+                allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+                allowfullscreen
+              ></iframe>
+            </div>
+            <div class="details">
+              <a
+                href="https://www.youtube.com/watch?v=jS4aFq5-91M&t=11017s"
+                target="_blank"
+              >
+                <p class="details-title">
+                  JavaScript Programming - Full Course
+                </p>
+              </a>
+              <p class="small">FreeCodecamp</p>
+              <div class="views">
+                <p>2.6M views</p>
+                <p>· 1 year ago</p>
+              </div>
+            </div>
+          </div>
+        </div>
```

Comment:
I will suggest you use appropriate semantics. You can wrap the entire **content-details** div in a **section** tag to indicate that it's a self-contained section of a content.